extended/95-background.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
What does 95 mean in the name?

Reply:
All the startup scripts are numbered to be run in order. "95" means "pretty close to the end but give the user a few slots (96-98) before the final script (99) runs.

Reply:
Thanks
since I noticed the pg version, for a minute I thought it was a pg matching version but the file was nothing to do with pg